<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputVariablesWidget</class>
 <widget class="QWidget" name="OutputVariablesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>461</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mVariablesGroupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Variables</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QCheckBox" name="mTHCheckBox">
        <property name="text">
         <string>target_heading</string>
        </property>
       </widget>
      </item>
      <item row="32" column="0">
       <widget class="QCheckBox" name="mTGPFCheckBox">
        <property name="text">
         <string>transmit_gain_propagation_factor</string>
        </property>
       </widget>
      </item>
      <item row="26" column="0">
       <widget class="QCheckBox" name="mSTCheckBox">
        <property name="text">
         <string>signal_at_target</string>
        </property>
       </widget>
      </item>
      <item row="30" column="0">
       <widget class="QCheckBox" name="mPFCheckBox">
        <property name="text">
         <string>propagation_factor</string>
        </property>
       </widget>
      </item>
      <item row="23" column="0">
       <widget class="QCheckBox" name="mSigPCheckBox">
        <property name="text">
         <string>signal_power</string>
        </property>
       </widget>
      </item>
      <item row="31" column="0">
       <widget class="QCheckBox" name="mRGPFCheckBox">
        <property name="text">
         <string>receive_gain_propagation_factor</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Geometry</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item row="24" column="1">
       <widget class="QCheckBox" name="mSlRCheckBox">
        <property name="text">
         <string>slant_range</string>
        </property>
       </widget>
      </item>
      <item row="22" column="1">
       <widget class="QCheckBox" name="mAACheckBox">
        <property name="text">
         <string>azimuth_angle</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1">
       <widget class="QCheckBox" name="mACheckBox">
        <property name="text">
         <string>angle</string>
        </property>
       </widget>
      </item>
      <item row="20" column="2">
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="23" column="1">
       <widget class="QCheckBox" name="mEACheckBox">
        <property name="text">
         <string>elevation_angle</string>
        </property>
       </widget>
      </item>
      <item row="22" column="2">
       <widget class="QCheckBox" name="mJSCheckBox">
        <property name="text">
         <string>jammer_to_signal</string>
        </property>
       </widget>
      </item>
      <item row="19" column="2">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Jamming</string>
        </property>
       </widget>
      </item>
      <item row="24" column="2">
       <widget class="QCheckBox" name="mNJPCheckBox">
        <property name="text">
         <string>noise_jammer_power</string>
        </property>
       </widget>
      </item>
      <item row="21" column="2">
       <widget class="QCheckBox" name="mJPCheckBox">
        <property name="text">
         <string>jammer_power</string>
        </property>
       </widget>
      </item>
      <item row="23" column="2">
       <widget class="QCheckBox" name="mJMDSCheckBox">
        <property name="text">
         <string>jammer_to_mds</string>
        </property>
       </widget>
      </item>
      <item row="28" column="2">
       <widget class="QCheckBox" name="mPJSCheckBox">
        <property name="text">
         <string>pulse_jammer_to_signal</string>
        </property>
       </widget>
      </item>
      <item row="29" column="2">
       <widget class="QCheckBox" name="mCJSCheckBox">
        <property name="text">
         <string>coherent_jammer_to_signal</string>
        </property>
       </widget>
      </item>
      <item row="27" column="2">
       <widget class="QCheckBox" name="mNJSCheckBox">
        <property name="text">
         <string>noise_jammer_to_signal</string>
        </property>
       </widget>
      </item>
      <item row="29" column="1">
       <widget class="QCheckBox" name="mTSAACheckBox">
        <property name="text">
         <string>target_to_sensor_azimuth_angle</string>
        </property>
       </widget>
      </item>
      <item row="25" column="2">
       <widget class="QCheckBox" name="mPJPCheckBox">
        <property name="text">
         <string>pulse_jammer_power</string>
        </property>
       </widget>
      </item>
      <item row="26" column="2">
       <widget class="QCheckBox" name="mCJPCheckBox">
        <property name="text">
         <string>coherent_jammer_power</string>
        </property>
       </widget>
      </item>
      <item row="30" column="1">
       <widget class="QCheckBox" name="mTSEACheckBox">
        <property name="text">
         <string>target_to_sensor_elevation_angle</string>
        </property>
       </widget>
      </item>
      <item row="30" column="2">
       <widget class="QCheckBox" name="mCJNCheckBox">
        <property name="text">
         <string>coherent_jammer_to_noise</string>
        </property>
       </widget>
      </item>
      <item row="28" column="1">
       <widget class="QCheckBox" name="mTSGRCheckBox">
        <property name="text">
         <string>target_to_sensor_ground_range</string>
        </property>
       </widget>
      </item>
      <item row="27" column="1">
       <widget class="QCheckBox" name="mTSSRCheckBox">
        <property name="text">
         <string>target_to_sensor_slant_range</string>
        </property>
       </widget>
      </item>
      <item row="26" column="1">
       <widget class="QCheckBox" name="mGACheckBox">
        <property name="text">
         <string>grazing_angle</string>
        </property>
       </widget>
      </item>
      <item row="25" column="1">
       <widget class="QCheckBox" name="mGRCheckBox">
        <property name="text">
         <string>ground_range</string>
        </property>
       </widget>
      </item>
      <item row="31" column="1">
       <widget class="QCheckBox" name="mTECheckBox">
        <property name="text">
         <string>terrain_elevation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sensor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="24" column="0">
       <widget class="QCheckBox" name="mNPCheckBox">
        <property name="text">
         <string>noise_power</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Detection</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Signal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="mPDCheckBox">
        <property name="text">
         <string>pd</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="mDTCheckBox">
        <property name="text">
         <string>detection_threshold</string>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QCheckBox" name="mSICheckBox">
        <property name="text">
         <string>signal_to_interference</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="mSHCheckBox">
        <property name="text">
         <string>sensor_heading</string>
        </property>
       </widget>
      </item>
      <item row="27" column="0">
       <widget class="QCheckBox" name="mRAGCheckBox">
        <property name="text">
         <string>receive_antenna_gain</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="mSAltCheckBox">
        <property name="text">
         <string>sensor_altitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="mTAltCheckBox">
        <property name="text">
         <string>target_altitude</string>
        </property>
       </widget>
      </item>
      <item row="29" column="0">
       <widget class="QCheckBox" name="mAFCheckBox">
        <property name="text">
         <string>attenuation_factor</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QCheckBox" name="mSNCheckBox">
        <property name="text">
         <string>signal_to_noise</string>
        </property>
       </widget>
      </item>
      <item row="28" column="0">
       <widget class="QCheckBox" name="mTAGCheckBox">
        <property name="text">
         <string>transmit_antenna_gain</string>
        </property>
       </widget>
      </item>
      <item row="25" column="0">
       <widget class="QCheckBox" name="mCPCheckBox">
        <property name="text">
         <string>clutter_power</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="mTRCheckBox">
        <property name="text">
         <string>target_roll</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="mTPCheckBox">
        <property name="text">
         <string>target_pitch</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="mSPCheckBox">
        <property name="text">
         <string>sensor_pitch</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="mSRCheckBox">
        <property name="text">
         <string>sensor_roll</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QCheckBox" name="mSSCheckBox">
        <property name="text">
         <string>sensor_speed</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="mRRCSCheckBox">
        <property name="text">
         <string>required_rcs</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QCheckBox" name="mSLatCheckBox">
        <property name="text">
         <string>sensor_latitude</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="mTLatCheckBox">
        <property name="text">
         <string>target_latitude</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="mTLonCheckBox">
        <property name="text">
         <string>target_longitude</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QCheckBox" name="mSLonCheckBox">
        <property name="text">
         <string>sensor_longitude</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="mTSCheckBox">
        <property name="text">
         <string>target_speed</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="mMSCheckBox">
        <property name="text">
         <string>masking_status</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="mRJPCheckBox">
        <property name="text">
         <string>required_jamming_power</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Signature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="mRSCheckBox">
        <property name="text">
         <string>radar_signature</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="mOSCheckBox">
        <property name="text">
         <string>optical_signature</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="mISCheckBox">
        <property name="text">
         <string>infrared_signature</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QCheckBox" name="mBRICheckBox">
        <property name="text">
         <string>background_radiant_intensity</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QCheckBox" name="mCRICheckBox">
        <property name="text">
         <string>contrast_radiant_intensity</string>
        </property>
       </widget>
      </item>
      <item row="19" column="3">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SAR</string>
        </property>
       </widget>
      </item>
      <item row="20" column="3">
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="21" column="3">
       <widget class="QCheckBox" name="mSARDTCheckBox">
        <property name="text">
         <string>sar_dwell_time</string>
        </property>
       </widget>
      </item>
      <item row="22" column="3">
       <widget class="QCheckBox" name="mSARRCheckBox">
        <property name="text">
         <string>sar_resolution</string>
        </property>
       </widget>
      </item>
      <item row="23" column="3">
       <widget class="QCheckBox" name="mSARCNCheckBox">
        <property name="text">
         <string>sar_clutter_to_noise</string>
        </property>
       </widget>
      </item>
      <item row="24" column="3">
       <widget class="QCheckBox" name="mSARDF">
        <property name="text">
         <string>sar_doppler_foldover</string>
        </property>
       </widget>
      </item>
      <item row="25" column="3">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
      <item row="26" column="3">
       <widget class="Line" name="line_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="27" column="3">
       <widget class="QCheckBox" name="mPCCheckBox">
        <property name="text">
         <string>pixel_count</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
